web/InsertCharacterAction.js

```
@@ -0,0 +1,423 @@
+    }
+  }
+};
+
+/**
+ * Add symbol elements to the "find by name" results container.
+ * @param {object} obj The object containing symbol results for the find by name query.
+ */
+InsertFromMenuAction.prototype.refreshSymbols_ = function (obj) {
+  goog.dom.removeChildren(this.foundByNameList_);
+  for (var code in obj) {
+    if (obj.hasOwnProperty(code)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)